### DISCUSSION TITLE
A new rzip-based compression program, modern-rzip

### DISCUSSION THREAD

**Discussion by pete4abw** [pete4abw/lrzip-next#115] [Ideas]
@kspalaiologos , who contributed **bzip3** to the `lrzip-next` project, has been working on `modern-rzip` [Modern Rzip](https://github.com/modern-rzip/modern-rzip) . It includes some new compression methods, **fast-lzma2** (@conor42 has helped in this project with assembler work), **tlsh**, and **ppmd**. It still uses **rzip** pre-processing. It's worth a peek!

**Comment by iczelia** [comment]
Hi! My project shares some common code with your `lrzip-next`, but I have decided to take the opportunity to remove some legacy code (as modern-rzip is a new project) that for backwards compatibility reasons could not be removed from `lrzip-next`. I have also re-tooled the build system to be a bit more lightweight.
Thanks for the inspiration!

**Reply by pete4abw** [reply]
I am happy that you are moving things forward. There's a lot of good in both `lrzip` and `lrzip-next`, but you are correct that some of the legacy code is a drag. I mainly keep it going for sentimental reasons and to keep my aging mind sharp ;-) . I'll follow your progress!
